easybuild/easyconfigs/d/dftd4/dftd4-3.7.0-foss-2023b.eb

```diff
@@ -0,0 +1,50 @@
+# A. Domingo (Vrije Universiteit Brussel)
+# J. Sassmannshausen (Imperial College London/UK)
+# C. Willemyns  (Vrije Universiteit Brussel)
+
+easyblock = 'CMakeNinja'
+
+name = 'dftd4'
+version = '3.7.0'
+
+homepage = 'https://dftd4.readthedocs.io'
+description = """
+The dftd4 project provides an implementation of the generally applicable, charge dependent
+London-dispersion correction, termed DFT-D4.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
```

Comment:
I think this could actually be dropped to `gfbf`, as it doesn't seem to use MPI. CREST itself can then also make use of gfbf. There were some previous versions that already used gfbf, but it seems like the most recent version (accidentally?) bumped the toolchain to foss.

https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/d/dftd4
https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/c/CREST